invenio_vocabularies/contrib/names/jsonschemas/names/name-v1.0.0.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "local://names/name-v1.0.0.json",
+  "description": "Names vocabulary.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "$ref": "local://definitions-v1.0.0.json#/$schema"
+    },
+    "id": {
+      "description": "URI or classification code as identifier - globally unique among all names schemes.",
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
+    "pid": {
+      "$ref": "local://definitions-v1.0.0.json#/internal-pid"
+    },
+    "scheme": {
+      "description": "Identifier of the name scheme.",
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
+    },
```

Comment:
The scheme is the "sub-vocabulary" or "vocabulary type". e.g. for subjects we have the global vocabulary, but then there are different inputs.
- Vocabulary: Subjects
    - Scheme: MeSH
    - Scheme: FOS
    - ...
 
Currently for affiliations it would be `RoR` indeed, and for ours `ORCiD`. Unless we do not want to distinguish different sources. It might make deduplication more complicated, but it already is. Knowing where the vocabulary items com from is useful IMO.

Reply:
To be taken into account when importing actual data in the system. Don't want to subtype/schemed it with `orcid`.